### DISCUSSION TITLE
stopwatch TMPE 2 ?

### DISCUSSION THREAD

**Discussion by Didoul35** [slyh/Cities2-TrafficLightsEnhancement#28] [Ideas]
Hello ! Great as a first mod! Do you think it would be possible to integrate a stopwatch like in TMPE?

You could be the successor to TMPE!

**Comment by slyh** [comment]
Do you mean a stopwatch icon on the screen?

**Reply by Didoul35** [reply]
I want say to you, a stopwatch for traffic light
